### PR TITLE
Add info about Ubuntu installation package

### LINKED ISSUE #295: Add info about Ubuntu installation package

### PR DESCRIPTION
**Pull request by pkuczynski**
Fixes #295

### CODE REVIEW

File: content/rvm/install.md [hunk 1]
```diff
@@ -4,16 +4,27 @@ title: Installing RVM
 
 # Installing RVM
 
-## Quick (guided) Install
+RVM supports most UNIX like systems and Windows (with Cygwin). The basic requirements are `bash`, `curl`, `gpg2` and overall GNU version of tools - but RVM tries to autodetect it and install anything that is needed.
```
Comment (mpapis) [inline content/rvm/install.md]: `Windows (with Cygwin)` - RVM also works perfectly fine with the new Ubuntu in Windows thingie, but I do not recall it's proper name.

File: content/rvm/install.md [hunk 2]
```diff
@@ -4,16 +4,27 @@ title: Installing RVM
...
-Before any other step install [mpapis](/authors/mpapis/) [public key](https://keybase.io/mpapis) (might need `gpg2`) (see [security](/rvm/security/))
+## Install GPG keys
+
+As a first step install [mpapis](/authors/mpapis/) [public key](https://keybase.io/mpapis) used to verify installation package to ensure [security](/rvm/security/).
 
     gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
     
-**Note: hkp://keys.gnupg.net can sometimes timeout you may want to use different keyserver
-- hkp://pgp.mit.edu
+***Note:*** *hkp://keys.gnupg.net can sometimes timeout you may want to use different keyserver, for example:*
```
Comment (mpapis) [inline content/rvm/install.md]: this note is still valid for `gpg` but if we fix the line above to use `gpg2` - a lot less users will hit this problem

File: content/rvm/install.md [hunk 3]
```diff
@@ -4,16 +4,27 @@ title: Installing RVM
...
+
+## Basic install
+
+### Ubuntu
+
+RVM have dedicated Ubuntu package, so please follow instructions posted here: https://github.com/rvm/ubuntu_rvm
+ 
+If you need a different (newer) version of RVM, after installing base version of RVM check the [Upgrading](/rvm/upgrading) section. 
 
-**Unless doing guided install you should read all sub-sections under the [RVM Section](/rvm/).**
```
Comment (mpapis) [inline content/rvm/install.md]: `+1000` - RVM should not require reading huge manual ;) our manuals should consist only of explanation of third party tools and concepts of RVM :)

File: content/rvm/install.md (hunk 2 above)
Comment (pkuczynski) [inline content/rvm/install.md]: this is just formatting change...

File: content/rvm/install.md (hunk 1 above)
Comment (pkuczynski) [inline content/rvm/install.md]: Ubuntu on Windows? Babun?

Comment (mpapis) [inline content/rvm/install.md]: https://www.howtogeek.com/265900/everything-you-can-do-with-windows-10s-new-bash-shell/

**Pull request merged**